src/elements/Typography/Typography.tsx

```diff
@@ -69,10 +75,10 @@ export interface TextProps
     SpaceProps,
     StyledSystemDisplayProps,
     TextAlignProps,
-    VerticalAlignProps {
+    VerticalAlignProps,
+    FontSizeProps,
+    LineHeightProps {
```

Comment:
#superminor but would you mind running alpha over these new interfaces?

Reply:
Done!